### PR TITLE
Filetypes

### PR DESCRIPTION
**Pull request by MasterVexillen**
Added code in main, metadata and motioncor for support for EER raw image filetype in motion correction

### CODE REVIEW

File: src/Ot2Rec/metadata.py [hunk 1]
```diff
@@ -84,34 +89,32 @@ def create_master_metadata(self):
...
+                                                       self.params['file_prefix'],
+                                                       self.params['filetype'])
             )
             
         if (len(raw_images_list) == 0):
             raise IOError("Error in Ot2Rec.metadata.Metadata.create_master_metadata: No vaild files found using given criteria.")
 
         # Convert potentially relative file paths to absolute paths
-        raw_images_list = [os.path.abspath(image) for image in raw_images_list]
+        raw_images_list = sorted([os.path.abspath(image) for image in raw_images_list])
```
Comment (elainehoml) [inline src/Ot2Rec/metadata.py:114]: I like that this is now sorted

Comment (elainehoml) [inline src/Ot2Rec/metadata.py:114]: I like that this is now sorted

**Review by elainehoml — APPROVED** [review]
All looks good to me! 👍 Small question on what happens to the dose data that you extracted (mc2.tmp) which is commented in `motioncorr.py`, which feeds into Issue #26 but happy to merge now if that's ok with you

File: src/Ot2Rec/motioncorr.py [hunk 2]
```diff
@@ -165,42 +167,49 @@ def _set_output_path(self):
...
+               '-Iter', str(self.params['MC2']['max_iterations']),
+               '-Group', '1' if self.params['MC2']['use_subgroups'] else '0',
+               '-FtBin', str(ftbin),
+               '-PixSize', str(self.params['MC2']['pixel_size']),
+               '-Throw', str(self.params['MC2']['discard_frames_top']),
+               '-Trunc', str(self.params['MC2']['discard_frames_bottom']),
         ]
+
+        if extra_info is not None:
+            cmd += ['-FmIntFile', 'mc2.tmp']
```
[the remark this answers is not in the document]
Comment (MasterVexillen) [reply src/Ot2Rec/motioncorr.py:210]: I haven't considered the clean-up for the moment so the temp file would be there at the end. It's only a one-liner for each image so wouldn't take up any space. 
However the information in the file comes from the metadata and the entire metadata is already recorded in the _mdout.yaml file (if the motioncor process terminates successfully).

**Pull request merged**